package.json

```diff
@@ -48,25 +49,23 @@
     "@babel/preset-env": "^7.6.2",
     "@babel/preset-react": "^7.0.0",
     "@babel/register": "^7.6.2",
+    "@cmfcmf/docusaurus-search-local": "^0.11.0",
+    "@docusaurus/core": "2.0.0-beta.18",
+    "@docusaurus/preset-classic": "2.0.0-beta.18",
+    "@mdx-js/react": "^1.6.22",
     "@rollup/plugin-buble": "^0.21.3",
     "@rollup/plugin-typescript": "^8.3.0",
-    "@testing-library/react": "^12.1.2",
-    "@testing-library/react-hooks": "8.0.1",
-    "@testing-library/user-event": "^13.5.0",
+    "@testing-library/react": "^13.0.0",
+    "@testing-library/user-event": "^14.0.0",
```

Comment:
With latest testing-library few test were breaking which works on browser. not sure if you faced same issue.

Reply:
Haven't noticed any issues like that so far, but I might have missed something.

There are a handful of tests that fail in `input_numeric_format.spec.js` using the updated test utils. The last time I checked (about a month ago), I could reproduce the failed tests in Chrome 113 and Safari 16.x.

Reply:
Cool lets try updating, I tried updating it last time I was fixing caret position issue, and got some issue so did not do upgrade.

Reply:
Sounds good. Will try and finish updating the remaining tests by the 18th.